<!DOCTYPE HTML>
<!--
	Hyperspace by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<!-- Global site tag (gtag.js) - Google Analytics -->
		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-80180575-2"></script>
		<script>
  			window.dataLayer = window.dataLayer || [];
  			function gtag(){dataLayer.push(arguments);}
  			gtag('js', new Date());
	  		gtag('config', 'UA-80180575-2');
		</script>
		<title>Genentech Sumedh Sankhe</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Header -->
			<header id="header">
				<a href="index.html" class="title">Genentech</a>
				<nav>
					<ul>
						<li><a href="index.html">Home</a></li>
						<li><a href="gene.html" class="active">Genentech</a></li>
                        <li><a href="ccis.html">Khoury College</a></li>
                        <li><a href="enel.html">Enel-X</a></li>
                        <li><a href="affect.html">Affect Mental Health</a></li>
					</ul>
				</nav>
			</header>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<section id="main" class="wrapper">
						<div class="inner">
							<div class="content">
                            	<h1 class="major">Principal Scientific Researcher</h1>
								<p>Considered the founder of the industry, Genentech, now a member of the Roche Group, has been 
                                    delivering on the promise of biotechnology for more than 40 years. Genentech is a 
                                    biotechnology company dedicated to pursuing groundbreaking science to discover and develop 
                                    medicines for people with serious and life-threatening diseases. Our transformational 
                                    discoveries include the first targeted antibody for cancer and the first medicine for 
                                    primary progressive multiple sclerosis.
                                </p>
							<h2>Peak Picking, Data QC and TargetedMSQC</h2>
								<p>Identifying quality peaks in any omics dataset is important not just identify existince of
                                    protein/peptides but also separate noise and signal. To be able to classify peak which not only
                                    of good intensity but also follow the rules of similarity, shape and other features as defined. 
									<br>
									MSnbase and XCMS two packges help deal with this problem in a rule based method which can be
                                    followed to an extent but manual intervention is required for more intensive tasks especially
                                    DIA data acquisition methods are used. Increasing sample sizes also demand the use of tools that 
                                    require minimal to no manual intervention and a quick turnaround time in terms human interpretable results
                                    through either visuals or summary tables.
                                </p>
								<div class = "row">
									<div class = "col-6" >
                                		<img src="images/msnbase_qc.png" width="100%">
									</div>
									<div class= "col-6">
										<img src="images/tmsqc.png" width="100%">
									</div>
								</div>
                                <p>TargetedMSQC was a package developed in-house at Genentech and published for through the paper 
                                    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173846/" class = "acive" target="_blank"">
                                    Quality assessment and interference detection in targeted mass spectrometry data using machine learning</a>
                                    the package enables us to use supervised machine learning methods to classify peak quality. My work involved 
                                    re-factoring the package, do feasibility study around extending a use case for DIA methods and develop an 
                                    interface to interact with the results without needing to code.
                                </p>
                                <h2>Large sample size and MSstats</h2>
                                <p>MSstats suite of statistical packages help lab based and data scientists analyse Mass Spec proteomics data generated by various tools like Spectronaut, Skyline, DiaNN to name a few. These tools
									work very efficiently on limited number of samples very well. One such analytical project involved analysing a CSF dataset with close to 150 samples with data collected using the DIA method resulting
									in close to 8000 proteins with multiple fragment ion, differing charge states and having a heavy (reference) and light (target) state. The resulting data file generated by intermediate tools very over
									~20GBs and a newer CSF study with 300 samples to ~105GBs.
									<br>
									<div class = 'row'>
										<img class = 'col-12' src="images/mstats_big.png" width="100%">
									</div>
									<br>
									The Image above shows a preliminary workflow to get around the memory limitations of using in memory computations. Stiching together a host of analytical and data manipulation pipelines to generate
									an analysis ready dataset. This initial workflow also resulted in a further collaboration with the Vitek Lab at Northeastern Univeristy, Boston. A new package called <a href = "https://bioconductor.org/packages/release/bioc/html/MSstatsBig.html"
									class = "active" target= "_blank">MSstatsBig</a> was developed to allow
									manipulation of out of memory datasets
								</p>
                                <h2>Publications, Collaborations & Posters</h2>
								<ul>
									<li>
										<a href="https://pubmed.ncbi.nlm.nih.gov/36370285/" class = "active" target = "_blank">Using SILAC to Develop Quantitative Data-Independent Acquisition (DIA) Proteomic Methods</a>
									</li>
									<li>
										<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10629259/" class="active" target="_blank">MSstats Version 4.0: Statistical Analyses of Quantitative Mass Spectrometry-Based Proteomic Experiments with Chromatography-Based
									Quantification at Scale
							   			</a>
									</li>
									<li>
										<a href="pdf/asms_poster_1.pdf" class = "active" target= "_blank">Using stable isotope labeling by amino acids in cell culture (SILAC) to 
									improve data independent acquisition (DIA) relative quantification</a>
									</li>
									<li>
										<a href="pdf/asms_poster_2.pdf" class = "active" target= "_blank">Identifying biomarkers of fibrosis from the urinary proteome of renal fibrosis patients using DIA mass spectrometry</a>
									</li>
								</ul>
								<h2>Recommendations</h2>
								<img class="image fit" src="images/gene-reco.png" >
							</div>
						</div>
					</section>

			</div>

		<!-- Footer -->
			<footer id="footer" class="wrapper alt">
				<div class="inner">
					<ul class="menu">
						<li>&copy; All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
					</ul>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>